python/res/config/config_settings.py

```diff
@@ -50,8 +50,10 @@ class ConfigSettings(BaseCClass):
 
 
 
-    def __init__(self , root_key):
-        c_ptr = ConfigSettings._alloc( root_key )
+    def __init__(self , root_key, c_ptr = None):
```

Comment:
No whitespaces between keyword_argument_name and their default value according to PEP8...
`c_ptr=None`

Reply:
fixed